meinberlin/assets/scss/components/_timeline.scss

```diff
@@ -118,22 +119,49 @@ $circle-size: 0.6em;
 }
 
 .phase-info {
+    box-shadow: 0 0 1.25em 0 rgba(0, 0, 0, 0.2);
```

Comment:
The button looks much smaller than in the design. I think we can increase it to `2em`.